scripts/generate_images.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
```

Comment:
Since every model's pipeline is quite specific, we probably won't be adding inference scripts to keep things more maintainable. Instead we're relying on example snippets on the main README and the (still WIP) documentation. 
If that makes sense, would you mind removing this file? :slightly_smiling_face:

Reply:
Sure thing, I have a side directory where I keep other scripts. Seeing generate_logits.py in scripts was the motivation to add it there

Reply:
Ah, I see! Yeah, we should remove generate_logits too, it's a rogue debugging script :smile: